docs/specs/servers/blockchain/blockchain-permissions-api.md

```diff
@@ -0,0 +1,180 @@
+# Blockchain API Sessions and Permissions
+
+## Sessions permissions storage
+
+### Get permissions list for account
+
+Used to get account list of active sessions
```

Comment:
We need either a new endpoint for revoked sessions, or including a field in the pci that includes the "status" ; else we cannot displayed previously revoked sessions on the ui